### PR TITLE
enhance rateplans with price

### PR DESCRIPTION
**Pull request by AWare**
this brings the catalog into the promo tool and performs gbp price lookups

ui work ongoing

conf file update required

### CODE REVIEW

File: app/conf/CatalogService.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.gu.zuora.{ZuoraApiConfig, rest}
+import com.typesafe.config.Config
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scalaz.std.scalaFuture._
+
+object CatalogService {
+  def fromConf(config: Config, env: String)={
```
Comment (paulbrown1982) [inline app/conf/CatalogService.scala]: Small style thing - best not to use abbreviations - so consider renaming to fromConfig 

* IMO - Config is an allowed abbreviation of Configuration just like Id is an allowed abbreviation of Identifier.

File: frontend/sass/main.sass [hunk 2]
```diff
@@ -136,4 +136,7 @@ md-datepicker.ng-pristine .md-datepicker-invalid
     width: 640px
 
 md-list-item.selected
-  background-color: rgba(170, 170, 170, 0.1)
\ No newline at end of file
+  background-color: rgba(170, 170, 170, 0.1)
+
+s
```
Comment (paulbrown1982) [inline frontend/sass/main.sass]: An 's' has sneaked in...

File: frontend/src/templates/RatePlanList.html [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
-        <md-checkbox ng-model="ratePlanIds[plan.ratePlanId]" aria-label="{{plan.ratePlanName}}" ng-repeat="plan in plans" ng-change="ctrl.applyRatePlans(ratePlanIds)">
-            {{plan.ratePlanName}}
-        </md-checkbox>
-    </div>
+  <h5>Rate plans</h5>
+  <div>
+    <md-checkbox ng-model="ratePlanIds[plan.ratePlanId]" aria-label="{{plan.ratePlanName}}" ng-repeat="plan in plans"
+                 ng-change="ctrl.applyRatePlans(ratePlanIds)">
+      {{plan.ratePlanName}}
+      <span ng-if="discount && plan.price">
```
Comment (paulbrown1982) [inline frontend/src/templates/RatePlanList.html]: 👍 design looks great here

File: frontend/sass/main.sass (hunk 2 above)
Comment (AWare) [inline frontend/sass/main.sass]: 🐍neaky 🐍nake

Comment (AWare) [inline frontend/sass/main.sass]: Oh, no this is the <s> element, which is a strikethrough showing no longer correct data.

Comment (AWare) [inline frontend/sass/main.sass]: the &gt;s&lt;

Comment (paulbrown1982) [inline frontend/sass/main.sass]: 😵😡

Comment (paulbrown1982) [inline frontend/sass/main.sass]: Add it to that SASS -> SCSS list so the braces can come back pls!

File: frontend/src/templates/RatePlanList.html [hunk 4]
```diff
@@ -5,8 +5,8 @@
                  ng-change="ctrl.applyRatePlans(ratePlanIds)">
       {{plan.ratePlanName}}
       <span ng-if="discount && plan.price">
-                <s>£{{plan.price | number: 2 }} </s> <md-icon md-font-set="material-icons">arrow_forward</md-icon>
-                <strong>£{{plan.price - (plan.price*discount/100) | number:2 }} </strong>
+                <s>{{plan.price | currency:"£" }} </s> <md-icon md-font-set="material-icons">arrow_forward</md-icon>
+                <strong>{{plan.price - (plan.price*discount/100) | currency:"£" }} </strong>
```
Comment (paulbrown1982) [inline frontend/src/templates/RatePlanList.html]: 👍

**Comment by AWare** [comment]
🎉 tested and verified in live

**Pull request merged**